gst/nnstreamer/tensor_split/gsttensorsplit.c

```diff
@@ -645,7 +645,12 @@ gst_tensor_split_get_property (GObject * object, guint prop_id,
         strings = (gchar **) g_ptr_array_free (arr, FALSE);
         p = g_strjoinv (":", strings);
         g_free (strings);
-        strv = g_strjoin (",", strv, p, NULL);
+        if (i > 0) {
+          strv = g_strjoin (",", strv, p, NULL);
```

Comment:
can we use g_strjoinv()? then we dont need the case of i>0.

Reply:
g_strjoinv() won't help. With g_strjoinv(), we'd need to make strv and p a str_array, which will do the same ugly string with strv="";

Reply:
g_strjoinv() does not insert separator at the end of the string if there is only 1 element in the array.
Yes, p will have to made str_array.

Reply:
"" != NULL. Thus, it will regard the initial strv ("") as yet another valid string.

Reply:
yes, you are right.